evap/staff/tests/test_importers.py

```diff
@@ -125,6 +125,7 @@ class TestEnrollmentImporter(TestCase):
     filename_valid = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data.xls")
     filename_valid_degree_merge = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data_degree_merge.xls")
     filename_valid_import_names = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data_import_names.xls")
+    filename_valid_consecutive_and_trailing_spaces = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data_consecutive_and_trailing_spaces.xls")
```

Comment:
not part of this PR, but do you think it might make sense to create some helper methods that create an (in-memory) xls from python data structures? We're adding more and more .xls files to the repo, which are not reviewable and must be manually edited in case the schema changes. we could keep one or two xls files in the tests to make sure we don't break reading "real" xls files. @JannisBerndt @janno42 @He3lixxx

Reply:
I think this makes sense. One could argue that the conversion is an additional point of failure but I think its not that critical.

Reply:
We could use csv of xls to make it human readable / reviewable. Schema changes would still be a problem.

I don't know whether using a data structure in code would boost maintainability. Are we talking about using a list of lists here?

Pylint organizes their tests like this: https://github.com/PyCQA/pylint/tree/master/tests/functional/r

Reply:
see #1503